frontend/lib/messages/components/message_bubble.component.dart

```diff
@@ -0,0 +1,114 @@
+import 'package:flutter/material.dart';
+import 'package:utm_marketplace/messages/model/message.model.dart';
+
+class MessageBubble extends StatelessWidget {
+  final Message message;
+  final String senderImageUrl;
+
+  const MessageBubble({
+    super.key,
+    required this.message,
+    required this.senderImageUrl,
+  });
+
+  String _formatTime(DateTime time) {
```

Comment:
Just an fyi: https://pub.dev/documentation/flutter_datetime_format/latest/

Probably best to leave this for now, and wait until you see how the backend provides timestamps and you can adjust then. But generally, you want to avoid manual date time formatting since there is usually a lot of intricacies with how date time is managed in code (best to leave that to libraries).